src/languageFeatures.ts

```diff
@@ -113,7 +113,9 @@ export function createDiagnosticsAdapter(
     }
   });
 
-  editor.getModels().forEach(onModelAdd);
+  for (const model of editor.getModels()) {
```

Comment:
I understand this is better, but was it causing any bug earlier that this had to be changed?

Reply:
I enabled `unicorn/no-for-each` in my ESLint configuration. It’s (almost) purely stylistic.